--- nfs-ganesha-3.0/src/scripts/ganeshactl/CMakeLists.txt.orig	2019-11-18 09:50:46.939249614 -0500
+++ nfs-ganesha-3.0/src/scripts/ganeshactl/CMakeLists.txt	2019-11-18 09:50:55.921421624 -0500
@@ -1,6 +1,6 @@
 # Building and packaging DBUS scripts for ganesha
 
-find_program(PYTHON NAMES "python3" "python")
+find_program(PYTHON NAMES "python3")
 
 # PyQT based class modules
 set(GANESHA_BASE_SRCS
