--- nfs-ganesha-2.5.0/src/CMakeLists.txt.orig	2017-06-19 10:53:31.929914300 -0400
+++ nfs-ganesha-2.5.0/src/CMakeLists.txt	2017-06-21 07:41:32.714553412 -0400
@@ -74,13 +74,13 @@
   elseif( ${SYS_RELEASE} MATCHES "Fedora" )
      message( STATUS "Detected a Linux Fedora machine" )
      set(DISTRO "FEDORA")
-  elseif( ${SYS_RELEASE} MATCHES "SLES" )
-     message( STATUS "Detected a Linux SLES machine" )
+  elseif( ${SYS_RELEASE} MATCHES "openSUSE Leap" )
+	  message( STATUS "Detected a Linux openSUSE machine" )
      set(DISTRO "SLES")
      set(LIBEXECDIR "/usr/lib")
   else( ${SYS_RELEASE} MATCHES "Red Hat" )
-     message( STATUS "Detected an UNKNOWN Linux machine" )
-     set(DISTRO "UNKNOWN")
+	  message( STATUS "Detected an UNKNOWN Linux machine: ${SYS_RELEASE}" )
+	  set(DISTRO "SLES")
   endif( ${SYS_RELEASE} MATCHES "Red Hat" )
 endif(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
 
